cdc/processor/pipeline/actor_node_context.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipeline
+
+import (
+	sdtContext "context"
+	"sync/atomic"
+
+	"github.com/pingcap/log"
+	"github.com/pingcap/ticdc/pkg/actor"
+	"github.com/pingcap/ticdc/pkg/actor/message"
+	"github.com/pingcap/ticdc/pkg/context"
+	"github.com/pingcap/ticdc/pkg/pipeline"
+	"go.uber.org/zap"
+)
+
+// send a tick message to actor if we get 32 pipeline messages
+const messagesPerTick = 32
+
+// actorNodeContext implements the NodeContext interface, with this we do not need to change too much logic to implement the table actor
```

Comment:
It is better that to limit the length of one line comment or code no more than 80 -- 100 chars.

Reply:
done